stock2shop/console_v2
# Handle Unauthorized request

If API response is 401. Navigate away and invalidate token.

To test:

1. Login as per normal
2. In a new tab login again into the v1 app https://console.stock2shop.com/, this should invalidate the API TOKEN
3. Click back to the v2 app. Perform any action that makes an api call, you should get logged out. This should happen automatically on any svelte-query screen i.e customer-detail
